mbtiles/src/copier.rs

```diff
@@ -662,11 +661,11 @@ impl MbtileCopierInt {
                 if idx > 0 {
                     sql.push_str(" OR\n");
                 }
-                writeln!(
-                    sql,
+                let filter = format!(
                     "(({prefix}tile_column * (1 << ({MAX_ZOOM} - {prefix}zoom_level))) BETWEEN {min_x} AND {max_x} \
-                     AND ({prefix}tile_row * (1 << ({MAX_ZOOM} - {prefix}zoom_level))) BETWEEN {min_y} AND {max_y})",
-                ).unwrap();
+                 AND ({prefix}tile_row * (1 << ({MAX_ZOOM} - {prefix}zoom_level))) BETWEEN {min_y} AND {max_y}))\n"
```

Comment:
not sure how this slipped in, good that CI caught it

```suggestion
                 AND ({prefix}tile_row * (1 << ({MAX_ZOOM} - {prefix}zoom_level))) BETWEEN {min_y} AND {max_y})\n"
```